docs/FAQs.md

```diff
@@ -0,0 +1,156 @@
+
+Relevant issues:  [[581](https://github.com/pytorch/serve/issues/581),[569](https://github.com/pytorch/serve/issues/569)]
+
+###  Are there any sample Models available?
+Various models are provided in Torchserve out of the box. Checkout out Torchserve [Model Zoo](https://github.com/pytorch/serve/blob/master/docs/model_zoo.md) for list of all available models in model zoo. Also check out examples folder all available [examples](https://github.com/pytorch/serve/tree/master/examples).
+
+### Does Torchserve has Windows Support?
+Currently, Torchserve supports Windows only via WSL(Windows Subsystem for Linux).
+The native Windows support will be added in upcoming releases.
+
+### Can I do streaming service with Torchserve like streaming speech recognition?
+Torchserve currently supports only inference through HTTP 1.0 - Request / Response style.
+
+### Is it possible to deploy model other than Pytorch framework?
+Yes it is possible to deploy model with custom python handler or you can use [AWS Multi Model Server](https://github.com/awslabs/multi-model-server) a similar model serving framework.
+NOTE : Torchserve has not been certified with any other framework apart from Pytorch.
+
+###  Does Torchserve support other models based on programming languages other than python?
+No, As of now only python based models are supported.
+
+
+### What benefits Torchserve has over AWS Multi-Model-server?
+Torchserve is derived from Multi-Model-server, But Torchserve is specifically tuned for Pytorch models. It also has new features like Snapshot and model versioning.
+
+## Deployment and config
+Relevant documents.
+- [Torchserve configuration](https://github.com/pytorch/serve/blob/master/docs/configuration.md)
+- [Model zoo](https://github.com/pytorch/serve/blob/master/docs/model_zoo.md#model-zoo)
+- [Snapshot](https://github.com/pytorch/serve/blob/master/docs/snapshot.md)
+- [Docker]([https://github.com/pytorch/serve/blob/master/docker/README.md](https://github.com/pytorch/serve/blob/master/docker/README.md))
+
+### Can I run Torchserve APIs on different ports other than 8080 & 8081?
+Yes, Torchserve API ports are configurable using a properties file or environment variable.Refer  [configuration.md](https://github.com/pytorch/serve/blob/master/docs/configuration.md) for more details.
+
+
+### How can I resolve model specific python dependency?
+You can provide a requirements.txt while creating a mar file using "--requirements-file/ -r" flag. Also, you can add dependency files using "--extra-files" flag. Refer  [configuration.md](https://github.com/pytorch/serve/blob/master/docs/configuration.md) for more details.
+
+### Can I deploy Torchserve in Kubernetes?
+Yes, you can deploy Torchserve in Kubernetes using Helm charts. Refer
+[Kubernetes deployment ](https://github.com/pytorch/serve/blob/master/kubernetes/README.md) for more details.
+
+### Can deploy Torchserve with ELB and ASG?
```

Comment:
@shivamshriwas Please note that TorchServe is cloud agnostic sol. When adding items related to AWS services, please prefix with AWS so that it is clear to the users what ELB/ASG and other services refer to

Reply:
made changes as suggested